src/Symfony/Component/Config/Tests/FileLocatorTest.php

```diff
@@ -86,7 +87,7 @@ public function testLocate()
     }
 
     /**
-     * @expectedException \InvalidArgumentException
+     * @expectedException FileLocatorFileNotFoundException
```

Comment:
PHP 7 requires a phpunit 5.* whereas phpunit 4.8 is the only option for PHP5.3 to 5.5.
Maybe phpunit 5.* handles `use`, but since we still support PHP 5.3, we can't use them.